#goal definition
float64[] TargetPoints
int32 NumTargetPoints
---
#result definition
bool AtTargetPoint
float64 errorToPoint
float64[] finalTargetPoint
---
#feedback
float64[] currentTarget
float64 distanceToNext